include/rocksdb/sst_file_reader.h

```diff
@@ -5,22 +5,60 @@
 #pragma once
 #ifndef ROCKSDB_LITE
 
-#include "rocksdb/sst_dump_tool.h"
-
+#include <inttypes.h>
 #include <memory>
 #include <string>
-#include "db/dbformat.h"
-#include "options/cf_options.h"
-#include "util/file_reader_writer.h"
+#include <vector>
+
+#include "rocksdb/env.h"
+#include "rocksdb/options.h"
+#include "rocksdb/table_properties.h"
+#include "rocksdb/types.h"
 
 namespace rocksdb {
 
+struct ImmutableCFOptions;
+class RandomAccessFileReader;
+class InternalKeyComparator;
+class TableReader;
+struct TableBuilderOptions;
+
+struct DefaultKvHandler {
+  DefaultKvHandler(bool output_hex) : output_hex_(output_hex) {}
+
+  void operator()(const Slice& key, const Slice& value, SequenceNumber sequence,
+                  unsigned char type) {
+    fprintf(stdout, "%s => %s, seq:%" PRIu64 ", type:%d\n",
```

Comment:
May nothing to do with the default handler and move the `printf` to the dump tool be better?  /cc @ajkr 

IMO, seem we don't need to expose these default Handle classes.

Reply:
Yeah that's a good point. Unless we wanna make these the default handlers for SstFileReader, which is fine with me too.